pkg/mcp/README.md

```diff
@@ -0,0 +1,168 @@
+# MCP (Model Context Protocol) Client
+
+This package provides functionality to interact with MCP (Model Context Protocol) servers from `kubectl-ai`.
+
+## Overview
+
+The MCP client allows `kubectl-ai` to connect to MCP servers, discover available tools, and execute them. This enables integration with various services and systems that expose their functionality through the MCP protocol.
+
+## Features
+
+- Connect to multiple MCP servers simultaneously
+- Automatic discovery of available tools from connected servers
+- Execute tools on remote MCP servers with parameter conversion
```

Comment:
is there an example of remote mcp server that we can try out ?

running command locally is already supported through custom tools, so good differentiation will be able to use it with a remote mcp server.